Belyashik/Belyashik/Program.cs

```diff
@@ -1,3 +1,25 @@
-﻿// See https://aka.ms/new-console-template for more information
+﻿using Test;
 
-Console.WriteLine("Hello, World!");
\ No newline at end of file
+var list = new MyList();
+
+// Exist
+list.Add(2);
+Console.WriteLine(list.Exist(2)); // true
+Console.WriteLine(list.Exist(3)); // false
+
+list.Clear();
+
+// AddRange
+list.AddRange(new []{11, 22, 33});
+Console.WriteLine(list.Get(0)); // 11
+Console.WriteLine(list.Get(1)); // 22
+Console.WriteLine(list.Get(2)); // 33
+
+list.Clear();
+
+// GetRange
+list.AddRange(new []{11, 22, 33, 44, 55});
```

Comment:
```suggestion
list.AddRange(new []{11, 22, 33, 44, 55});
```

Говорят хорошо было бы пробелы проставлять когда надо